src/MSALWrapper/AuthFlow/Broker.cs

```diff
@@ -153,7 +142,7 @@ protected override async Task<(TokenResult, IList<Exception>)> GetTokenInnerAsyn
                 this.errors.Add(ex);
             }
 
-            return (null, this.errors);
+            return (tokenResult, this.errors);

```

Comment:
I think that we could just return `tokenResult` and modify the `AuthFlowBase` class to always return `this.errors`. This is a separate change though, not entirely related to this PR, so I could see it done in a follow up.

Reply:
Discussed offline - we like this idea, but will address in a separate PR.